src/content/cloud/credentials.mdx

```diff
@@ -100,4 +100,4 @@ For example, you could use `kubectl` to deploy our `hello-world` application:
 kubectl apply -f https://raw.githubusercontent.com/okteto/go-getting-started/master/k8s.yml
 ```
 
-> As mentioned before, [credential plugins](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#client-go-credential-plugins) are a way of managing Kubernetes cluster credentials in a more secure way as you get time-bound tokens for your service account instead of using auto-generated secret-based ones. In case you face any issue with Okteto's credential plugin or you just want to keep using static auto-generated tokens, you can do it by setting the environment variable `OKTETO_USE_STATIC_KUBETOKEN` to `true` before running any Okteto command. Be aware that using those static tokens is not recommended by Kubernetes, and you will start getting warnings in your `kubectl` output from version `1.27` encouraging you to stop using them.
\ No newline at end of file
+> As mentioned before, [credential plugins](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#client-go-credential-plugins) are a way of managing Kubernetes cluster credentials in a more secure way as you get time-bound tokens for your service account instead of using auto-generated secret-based ones. In case you face any issue with Okteto's credential plugin, or you just want to keep using static auto-generated tokens, you can do it by setting the environment variable `OKTETO_USE_STATIC_KUBETOKEN` to `true` before running any Okteto command. Be aware that using those static tokens is not recommended by Kubernetes, and you will start getting warnings in your `kubectl` output from version `1.27` encouraging you to stop using them.
```

Comment:
```suggestion
[Credential plugins](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#client-go-credential-plugins) are a way of managing Kubernetes cluster credentials in a more secure way as you get time-bound tokens for your service account instead of using auto-generated secret-based ones. If you prefer to use static, auto-generated tokens you can do so by setting the environment variable `OKTETO_USE_STATIC_KUBETOKEN` to `true` before running any Okteto command. Be aware that using static tokens is not recommended by Kubernetes and you will receive warnings in your `kubectl` output from version `1.27` onward encouraging you to stop using them.
```